helpers/api.js

```diff
@@ -0,0 +1,4 @@
+exports.getBaseUrl = () => {
+  const env = process.env.DDENV === "prod" ? "" : "_" + process.env.DDENV;
```

Comment:
should we default to prod so customers don't need a `.env` file?

Reply:
+1. we don't want the user having to configure this if possible

Reply:
This was originally a two step thing, converted to process.env would look like
```javascript
  const env = process.env.DDENV || "prod";  // default to prod
  const envUrlPart = env === "prod" ? "" : "_" + env;
// use envUrlPart in format string
```

Reply:
@joetempleman @jryoo Good catch - fixed